### PR TITLE
Patryk database

### PR DESCRIPTION
**Pull request by Hinren**
database update

### CODE REVIEW

File: Clinic_project.Database/Table/Medical_leave.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[Medical_leave]
+(
+	[Id] INT NOT NULL PRIMARY KEY IDENTITY(1,1), 
+    [id_users] INT NULL FOREIGN KEY REFERENCES Users(Id), 
+    [id_doctors] INT NULL FOREIGN KEY REFERENCES doctors(Id), 
+    [from] DATETIME NULL, 
+    [down] DATETIME NULL,
```
Comment (Seremontis) [inline Clinic_project.Database/Table/Medical_leave.sql]: maybe rename on "to" or "end"

File: Clinic_project.Database/Table/Users.sql [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿CREATE TABLE [dbo].[Users]
+(
+	[Id] INT NOT NULL PRIMARY KEY IDENTITY(1,1), 
+    [first_name] NVARCHAR(50) NULL, 
+    [last_name] NVARCHAR(50) NULL,
```
Comment (Seremontis) [inline Clinic_project.Database/Table/Users.sql]: What if surname is longer than 50 chars ?

File: Clinic_project.Database/Table/Users.sql [hunk 3]
```diff
@@ -0,0 +1,15 @@
+﻿CREATE TABLE [dbo].[Users]
```
Comment (Seremontis) [inline Clinic_project.Database/Table/Users.sql]: Maybe, Will we make two tables? Doctor and client separately

File: Clinic_project.Database/Table/Users.sql [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+(
+	[Id] INT NOT NULL PRIMARY KEY IDENTITY(1,1), 
+    [first_name] NVARCHAR(50) NULL, 
+    [last_name] NVARCHAR(50) NULL, 
+    [email] NVARCHAR(50) NULL, 
+    [street] NVARCHAR(50) NULL, 
+    [city] NVARCHAR(35) NULL, 
+    [gender] NVARCHAR(50) NULL, 
+    [password] NVARCHAR(50) NULL,
```
Comment (Seremontis) [inline Clinic_project.Database/Table/Users.sql]: Here, We will use hash code chars. We have to check how many chars require to hashes

File: Clinic_project.Database/Table/Users.sql [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+(
+	[Id] INT NOT NULL PRIMARY KEY IDENTITY(1,1), 
+    [first_name] NVARCHAR(50) NULL, 
+    [last_name] NVARCHAR(50) NULL, 
+    [email] NVARCHAR(50) NULL, 
+    [street] NVARCHAR(50) NULL, 
+    [city] NVARCHAR(35) NULL, 
+    [gender] NVARCHAR(50) NULL, 
+    [password] NVARCHAR(50) NULL, 
+    [pesel] NVARCHAR(50) NULL,
```
Comment (Seremontis) [inline Clinic_project.Database/Table/Users.sql]: nvarchar(12)

File: Clinic_project.Database/Table/doctors.sql [hunk 6]
```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[doctors]
+(
+	[Id] INT NOT NULL PRIMARY KEY IDENTITY(1,1), 
+    [first_name] NVARCHAR(50) NULL, 
+    [last_name] NVARCHAR(50) NULL, 
+    [special_identinty] NVARCHAR(50) NULL
+)
```
Comment (Seremontis) [inline Clinic_project.Database/Table/doctors.sql]: add table with data visited 
example: cause visit or oder code required to fill L4

File: Clinic_project.Database/Table/Workplace_nip.sql [hunk 7]
```diff
@@ -0,0 +1,6 @@
+﻿CREATE TABLE [dbo].[Workplace_nip]
```
Comment (Seremontis) [inline Clinic_project.Database/Table/Workplace_nip.sql]: name clinic and address? public or private clinic

**Comment by Hinren** [comment]
git

**Pull request merged**